### ISSUE TITLE
[Develop] JS related issues

### ISSUE THREAD

**Issue by medariox** [pymedusa/Medusa#1353]
TODO: 
- [x] [File browser: close button is showing "close" together with the "x"](https://github.com/pymedusa/Medusa/issues/1353#issuecomment-257041472)
- [ ] [Rename page issue](https://github.com/pymedusa/Medusa/issues/1347) (**Only Windows?**)
- [x] [Parent Folder is not being saved, when clicked "set as default"](https://github.com/pymedusa/Medusa/issues/1424)
- [x] [Repeated checks for news? check_for_new_news loops](https://github.com/pymedusa/Medusa/issues/1348)
- [x] [Button "Checkout button" not working. Calls nothing. Nothing in logs](https://github.com/pymedusa/Medusa/issues/1353#issuecomment-257041667)
- [x] After clicking on the search icon for an episode, status never changes from "success" without reloading page
- [x] displayShow no longer has buttons to hide / show seasons: 
Tested this: 1. disable show all seasons. No buttons. 2. restarted. Buttons. 3. enabled show all seasons. No buttons. 4. disabled show all seasons. No buttons. (so I gues at this point I need to restart again) (**Works after restart.**)

**Comment by labrys** [comment]
* Repeated checks for github version
* After clicking on the search icon for an episode, status never changes from "success" without reloading page

**Comment by medariox** [comment]
- Repeated checks for github version
@labrys That should be fixed in latest develop version.

**Comment by fernandog** [comment]
File browser: close button is showing "close" together with the "x"
![fb](https://cloud.githubusercontent.com/assets/2620870/19824015/f9f0162a-9d4a-11e6-82e7-80329f86066a.png)

**Comment by fernandog** [comment]
Button "Checkout button" not working. Calls nothing. Nothing in logs
![gc](https://cloud.githubusercontent.com/assets/2620870/19824045/1a62efd6-9d4b-11e6-80a9-086f64851329.png)

@ratoaq2

**Comment by OmgImAlexis** [comment]
A lot of these are from updating jquery so these were expected. Will be fixed Tuesday night once I've got internet again.

**Comment by fernandog** [comment]
this issues are in master too then?!

**Comment by labrys** [comment]
@OmgImAlexis your url: http://chandrewz.github.io/blog/authentication-with-vue-router

**Comment by Thraxis** [comment]
Note:  These are all issues with 0.17 currently

Tables broken, No Sort or Filter on:
home
schedule
history
displayShow
etc.

displayShow no longer has buttons to hide / show seasons
Search returns a json response page on displayShow

buttons do not work on Manual Search page

**Comment by fernandog** [comment]
@pymedusa/developers if all this is happening on `master` we should revert jquery upgrade and hotfix.

**Comment by jxer** [comment]
i have failures all over in 2 servers, both are showing 0 shows and 1 doesn't even start after the upgrade, you need to fall back to the last commit. i am completely offline at this point

**Comment by OmgImAlexis** [comment]
Okay that doesn't sound at all like something that could have been caused by what I did since I only edited a handful of lines and updated jquery. 

Do any of you have any errors showing in medusa or the js console?

**Comment by Thraxis** [comment]
vender.min.js?200:2 jQuery.Deferred exception: Cannot read property 'settings' of undefined TypeError: Cannot read property 'settings' of undefined
    at Object.MEDUSA.common.init (http://unraid:9081/js/common/init.js?200:92:18)
    at Object.exec (http://unraid:9081/js/core.js?200:26:35)
    at HTMLDocument.init (http://unraid:9081/js/core.js?200:34:14)
    at l (http://unraid:9081/js/vender.min.js?200:2:8756)
    at m (http://unraid:9081/js/vender.min.js?200:2:9071) undefined

**Comment by ratoaq2** [comment]
@jxer can you check this #1364 ?

If that information couldn't help you, please reopen #1364 with information of your .db files in the application folder.

**Comment by OmgImAlexis** [comment]
@Thraxis fixed in #1365 and tooltips fixed in #1366

**Comment by p0psicles** [comment]
I don't think we should talk about reverting anymore.  With all the help we're getting,  we have these fixed in no time. For users on develop,  just update on a daily base and keep us updated like you have been. 

Most changes like the jquery have been done to remove technical debt.  So they had to be done at some point.

**Comment by p0psicles** [comment]
All of the config related issues should be resolved now. I don't really have time left to check them case by case. So for the ones who reported them. Please update the issue.

**Comment by OmgImAlexis** [comment]
@p0psicles I'm guessing I accidentally removed the config page js because of that change? Lol
I might move all the pages to MEDUSA.pages.x.x to stop that happening again.

**Comment by Thraxis** [comment]
@xo you changed line 98 in ajax-episode-search.js from 'success' to 'done',  now when a forced search is finished it never removes the spinner and sets the icon back to the search image.   was that a mistaken change?  https://github.com/pymedusa/Medusa/blob/d36e50ca8a36cd14d3b78146e3d484235489c955/static/js/ajax-episode-search.js#L96-L103

**Comment by fernandog** [comment]
@xo any ETA to fix this issues? thanks!

**Comment by bmehling** [comment]
Please note: the user @OmgImAlexis uses an avatar w/ "X O". The user @xo mentioned here is not a contributor to this project. Thx.

**Comment by OmgImAlexis** [comment]
@bmehling okay?

**Comment by OmgImAlexis** [comment]
@fernandog will try and fix these all tonight.

**Comment by OmgImAlexis** [comment]
Can someone confirm the hide/show season button is working on develop d36e50ca8a36cd14d3b78146e3d484235489c955?

**Comment by fernandog** [comment]
@xo it works for me but @medariox should confirm as it wasnt working for him

**Comment by OmgImAlexis** [comment]
@fernandog so checkout works now or only in master?

**Comment by fernandog** [comment]
@omgImAlexis didnt see any fix. Still broken in develop. Try it

**Comment by OmgImAlexis** [comment]
1. Rename page issue seems to be Windows only and can be left for now as it doesn't really break anything.
2. File browser button is a issue with updating some of the libs and I'll fix that.
3. Checkout button seems to work fine in b0731a187adeb66ed37f2162f4c7b564d42ad995 but should really show a dialog or something to the user.
4. displayShow buttons "work" but currently require a restart which I guess is okay for now but I do want to fix it soon.

**Comment by duramato** [comment]
Unckecking the proper search hides the **Entire** remaining search settings
![sad](https://lol.doyou.bid/UtrBT.png)
![dde](https://lol.doyou.bid/zAgKW.png)

**Comment by OmgImAlexis** [comment]
@duramato Try the `fix-proper-div` branch.

**Comment by OmgImAlexis** [comment]
@medariox can you open a new issue with just the remaining things?

**Comment by fernandog** [comment]
@xo still waiting for the fix : File browser: close button is showing "close" together with the "x"

**Comment by OmgImAlexis** [comment]
Just remove the close text and it'll be fixed. Super simple.

**Comment by duramato** [comment]
Filter on History page on the Episode column doesn't work if it has the numbering like`White Rabbit Project - S01E10`, `White Rabbit Project` only works. On chrome

**Comment by OmgImAlexis** [comment]
Once https://github.com/pymedusa/Medusa/pull/1807 is merged I'll fix the close issue and then can someone open a new issue for the rename page.

**Comment by OmgImAlexis** [comment]
Closing this as the last issue is fixed in the above PR, the renaming bug has it's own issue open.
